relay/purger/purger.go

```diff
@@ -19,21 +19,29 @@ import (
 	"sync"
 	"time"
 
-	"github.com/pingcap/dm/pkg/log"
 	"github.com/pingcap/errors"
 	"github.com/siddontang/go/sync2"
+	"go.uber.org/zap"
 
 	"github.com/pingcap/dm/dm/pb"
+	"github.com/pingcap/dm/pkg/log"
 	"github.com/pingcap/dm/pkg/streamer"
 	"github.com/pingcap/dm/pkg/utils"
 )
 
 // errors created by Purger
 var (
+	// logger writes log start with `[component=purger]`
+	logger log.Logger
+
 	ErrSelfPurging  = errors.New("this strategy is purging")
 	MsgOtherPurging = "%s is purging"
 )
 
+func init() {
+	logger = log.With(zap.String("component", "purger"))
```

Comment:
Is creating `logger` in `init` fine? does global `appLogger` initialized when calling `init`?

Reply:
@WangXiangUSTC comment for you

Reply:
modified